src/sharedData/visualization/components/VisualizationWrapper.js

```diff
@@ -83,6 +85,15 @@ const VisualizationWrapper = props => {
 
   useBreadcrumbsParams(useMemo(() => ({ mapTitle }), [mapTitle]));
 
+  useSocialShareContext(
```

Comment:
@shrouxm Yes, that is an error, if there is an instance of `<SocialShare>` receiving name as a parameter. This is how Breadcrumbs and social share and Container are working right now, so that was the best way to inform components outside of the current route about the context of it. Let me know if you have a better way of handling this, I think we could handle that on a separate Issue if we find a better way of doing this.

Reply:
ah, got it

i'm not sure exactly how our routes work, but the main alternative implementation path that comes to mind would be turning the current app scaffold code (app bar, breadcrumbs, social share, etc) into a component. the component would accept parameters for what we're currently using `useContainerContext`, `useSocialShareContext`, and `useBreadcrumbsParams`. and then the outermost element of each route would be an instance of that component, with the old route as its child. i don't know our codebase well enough to say whether that would be preferable

but especially given that `useBreadcrumbsParams` already worked this way i agree this would be a separate issue, so i will file that and discussion can continue there

Reply:
https://github.com/techmatters/terraso-web-client/issues/725